### PR TITLE
temporarily remove RAL stratum 1 due to connectivity issues

### PR DESCRIPTION
**Pull request by vvolkl**

### CODE REVIEW

File: etc/cvmfs/domain.d/cern.ch.conf [hunk 1]
```diff
@@ -6,7 +6,7 @@ CVMFS_FALLBACK_PROXY="$CVMFS_FALLBACK_PROXY"
...
 # Users can turn it on the default.local file
 if [ "$CVMFS_USE_CDN" = "yes" ]; then
-    CVMFS_SERVER_URL="http://s1cern-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1ral-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1bnl-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1fnal-cvmfs.openhtc.io:8080/cvmfs/@fqrn@;http://s1ihep-cvmfs.openhtc.io:8080/cvmfs/@fqrn@;http://s1swinburne-cvmfs.openhtc.io:8080/cvmfs/@fqrn@;http://s1sampa-cvmfs.openhtc.io:8080/cvmfs/@fqrn@"
+    CVMFS_SERVER_URL="http://s1cern-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1bnl-cvmfs.openhtc.io/cvmfs/@fqrn@;http://s1fnal-cvmfs.openhtc.io:8080/cvmfs/@fqrn@;http://s1ihep-cvmfs.openhtc.io:8080/cvmfs/@fqrn@;http://s1swinburne-cvmfs.openhtc.io:8080/cvmfs/@fqrn@;http://s1sampa-cvmfs.openhtc.io:8080/cvmfs/@fqrn@"
 else
     CVMFS_SERVER_URL="http://cvmfs-stratum-one.cern.ch:8000/cvmfs/@fqrn@;http://cernvmfs.gridpp.rl.ac.uk:8000/cvmfs/@fqrn@;http://cvmfs-s1bnl.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1fnal.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-stratum-one.ihep.ac.cn:8000/cvmfs/@fqrn@;http://cvmfs-s1.hpc.swin.edu.au:8000/cvmfs/@fqrn@;http://sampacs01.if.usp.br:8000/cvmfs/@fqrn@"
```
Comment (DrDaveD) [inline etc/cvmfs/domain.d/cern.ch.conf:11]: Also remove cernvmfs.gridpp.rl.ac.uk in all the else cases.

**Review by DrDaveD — CHANGES_REQUESTED** [review]

**Review by jose-caballero — COMMENTED** [review]
the actual URLs for the RAL Stratum-1 are
cvmfs-egi.gridpp.rl.ac.uk
cernvmfs.gridpp.rl.ac.uk

In general, any URL with .gridpp.rl.ac.uk in it

**Comment by vvolkl** [comment]
OK, thank you. Should have now gotten all of them.

**Review by jose-caballero — APPROVED** [review]

**Review by DrDaveD — APPROVED** [review]

**Comment by DrDaveD** [comment]
I'll leave it to you to merge & deploy.  I'll make cherry-picks for egi and osg distributions

**Pull request merged**